packages/phase2cli/src/lib/constants.ts

```diff
@@ -0,0 +1,139 @@
+import chalk from "chalk"
+import logSymbols from "log-symbols"
+import emoji from "node-emoji"
+
+/** Theme */
+export const theme = {
+    yellow: chalk.yellow,
+    magenta: chalk.magenta,
+    red: chalk.red,
+    green: chalk.green,
+    underlined: chalk.underline,
+    bold: chalk.bold,
+    italic: chalk.italic
+}
+
+export const symbols = {
+    success: logSymbols.success,
+    warning: logSymbols.warning,
+    error: logSymbols.error,
+    info: logSymbols.info
+}
+
+export const emojis = {
+    tada: emoji.get("tada"),
+    key: emoji.get("key"),
+    broom: emoji.get("broom"),
+    pointDown: emoji.get("point_down"),
+    eyes: emoji.get("eyes"),
+    wave: emoji.get("wave"),
+    clipboard: emoji.get("clipboard"),
+    fire: emoji.get("fire"),
+    clock: emoji.get("hourglass"),
+    dizzy: emoji.get("dizzy_face"),
+    rocket: emoji.get("rocket"),
+    oldKey: emoji.get("old_key"),
+    pray: emoji.get("pray"),
+    moon: emoji.get("moon"),
+    upsideDown: emoji.get("upside_down_face"),
+    arrowUp: emoji.get("arrow_up"),
+    arrowDown: emoji.get("arrow_down")
+}
+
+/** ZK related */
+export const potDownloadUrlTemplate = `https://hermez.s3-eu-west-1.amazonaws.com/`
```

Comment:
Does this mean we have to pre upload pTau files to and endpoint with this shape? Do we host this?

Reply:
This endpoint is related to the hosted powers of tau files from the Trusted Setup done for Hermez (https://github.com/iden3/snarkjs#7-prepare-phase-2). Our interaction with this external resource for downloading ptau files can occur optionally during execution of the `setup` command. 

If the coordinator were to set up a ceremony with circuits for which it does not have the correct powers of tau file on the machine or the files are not adequate in terms of amount of powers (to support the constraints of the circuits choosen for the setup). In these cases, one (or more, if needed) ptau file will be downloaded from the endpoint. Our AWS S3 storage infrastructure will store all ceremony files when the setup is done correctly; from .zkey to .r1cs files, and def. the .ptau are included to making everything verifiable during (and at the end) of the ceremony.

LMK if this makes sense to you and if I have answered your question correctly. Thanks!